tensorboard/webapp/metrics/views/card_renderer/scalar_card_fob_controller.ts

```diff
@@ -74,18 +74,18 @@ export class ScalarCardFobController {
     return this.scale.forward(
       this.minMaxHorizontalViewExtend,
       [0, this.axisSize],
-      this.timeSelection.start.step
+      this.timeSelection?.start.step ?? this.minMaxStep.minStep
```

Comment:
why don't we return empty string if timeSelection is undefined?

Reply:
Empty string is not a valid argument for that function.

![Screenshot from 2022-10-31 09-46-45](https://user-images.githubusercontent.com/78179109/199062867-8ff66d55-21c2-46a4-9b3b-ffca2569ebfd.png)

Reply:
oh I mean return empty string for `getAxisPositionFromStartStep` then here will be just `this.timeSelection!.start.step`?